### PR TITLE
[WIP] Quotas

### PR DESCRIPTION
**Pull request by jmcarp**
Emit current quota per org--should probably be run daily / hourly. Todo: aggregate daily / hourly quotas to monthly.

Includes #2

cc @jcscottiii

### CODE REVIEW

File: poll_quotas.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    )
+    token.raise_for_status()
+
+    return OAuth2Session(client_id, token=token.json())
+
+
+def poll_quotas(session, client, poll_index, doc_type):
+    now = datetime.datetime.now()
+    orgs = {
+        org['entity']['quota_definition_guid']: org
```
Comment (cnelson) [inline poll_quotas.py]: Won't we lose orgs here if someone sets multiple orgs to use the same quota?

**Comment by jmcarp** [comment]
Added some basic integration tests, since the code is mostly glue, like the diego aggregation. WDYT @cnelson ?

**Comment by jmcarp** [comment]
These changes LGTM 🍰

**Comment by cnelson** [comment]
@jmcarp This now copies the daily poll documents for the month into the aggregate doc for each org WDYT?

**Pull request merged**